src/constants/Forms.js

```diff
@@ -1,39 +1,50 @@
-export const FIELDS_SIGNUP = [
-  {
-    id: "first_name",
-    label: "Nombre",
-    type: "text",
+// import { rulesFields } from "./RulesFieldsUsers";
+const formFields = {
+  email: {
+    label: "Email",
+    name: "email",
+    type: "email",
   },
-  {
-    id: "last_name",
+
+  last_name: {
     label: "Apellido",
+    name: "last_name",
     type: "text",
   },
-  {
-    id: "email",
-    label: "Email",
-    type: "email",
+
+  name: {
+    label: "Nombre",
+    name: "first_name",
+    type: "text",
   },
-  {
-    id: "password",
+
+  password: {
     label: "Password",
-    type: "Password",
+    name: "password",
+    type: "text",
   },
-  {
-    id: "password_confirmation",
+
+  password_confirmation: {
```

Comment:
```suggestion
  passwordConfirmation: {
```
Use Camel case on variable names